helm-charts/yunikorn/templates/admission.yaml

```
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: yunikorn-admission-controller
+  labels:
+    app: yunikorn
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: yunikorn
+  template:
+    metadata:
+      labels:
+        app: yunikorn
+    spec:
+      serviceAccountName: {{ .Values.admission_controller_image.serviceAccountName }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- range .Values.imagePullSecrets }}
+        - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      hostNetwork: true
+      tolerations:
+        - operator: "Exists"
+      affinity:
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: component
+                    operator: In
+                    values:
+                      - yunikorn-scheduler
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: yunikorn-admission-controller-webhook
+          image: {{ .Values.admission_controller_image.repository }}:{{ .Values.admission_controller_image.tag }}
+          imagePullPolicy: {{ .Values.admission_controller_image.pullPolicy }}
+          ports:
+            - containerPort: 9089
+              name: webhook-api
+          resources:
+            limits:
+              cpu: "500m"
+              memory: 500Mi
+            requests:
+              cpu: "100m"
+              memory: 500Mi
+          volumeMounts:
+            - name: webhook-tls-certs
+              mountPath: /run/secrets/tls
+              readOnly: true
+          env:
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POLICY_GROUP
+              value:  {{ .Values.admission_controller_image.policyGroup }}
```

```suggestion
              value:  {{ .Values.admissionControllerImage.policyGroup }}
```
Minor nit: can we take this opportunity to change the name of this variable to use camelcase, conforming to the Helm [best practice](https://helm.sh/docs/chart_best_practices/values/#naming-conventions)?

---

That sounds like a real good idea